openreview/conference/helpers.py

```diff
@@ -85,6 +94,17 @@ def get_conference_builder(client, request_form_id, support_user='OpenReview.net
     if isinstance(submission_additional_options, str):
         submission_additional_options = json.loads(submission_additional_options.strip())
 
+    if submission_second_due_date:
```

Comment:
can we do this inside of the builder?, I guess we already do this.

Reply:
here: https://github.com/openreview/openreview-py/blob/master/openreview/conference/builder.py#L1281

if second_due_date exists then content['pdf']['required'] = False

Reply:
make sure 'pdf' exists in content